functions/kontent_ai_webhook_generic.ts

```diff
@@ -43,7 +44,10 @@ export const handler: Handler = async (event) => {
   }
 
   // download the affected items
-  const deliveryClient = new DeliveryClient({ projectId });
+  const deliveryClient = new DeliveryClient({
+    projectId,
+    globalHeaders: () => [{ header: 'X-KC-SOURCE', value: `${packageJson.name};${packageJson.version}` }]
```

Comment:
I might add the parameter with underscore to see that original headers are present in `_headers`

Reply:
https://github.com/kontent-ai/integration-example-recombee/pull/6#discussion_r1073425158